docs/CHANGELOG.asciidoc

```diff
@@ -33,6 +34,68 @@ This section summarizes the changes in each release.
+== {kib} 7.6.1
+
+See <<breaking-changes-7.6,breaking changes in 7.6>>.
+
+[float]
+[[enhancement-7.6.1]]
+=== Enhancements
+SIEM::
+* Imports rules unit tests {pull}57466[#57466]
+
+[float]
+[[bug-7.6.1]]
+=== Bug fixes
+APM::
+* Fixes cloud env in APM tutorial {pull}57817[#57817]
+* Adds `xpack.apm.enabled` key to config schema {pull}57539[#57539]
+* X-axis labels on Error occurrences chart are incorrect based on {kib} timezone {pull}55686[#55686]
+Canvas::
+*  Sanitizes workpad before sending to API {pull}57704[#57704]
+Lens::
+* Fixes bugs in Lens filters (#56441) {pull}56648[#56648]
+* Makes field stats work for index patterns without time fields {pull}56759[#56759]
+Machine Learning::
+* Fixes Data Visualizer responsive layout  {pull}56372[#56372]
+* Fixes overall stats for saved search on the Data Visualizer page {pull}57312[#57312]
+* Fixes jobs list default refresh {pull}57086[#57086]
+* Updates schema definition for create route {pull}56979[#56979]
+* Fixes brush visibility. {pull}57564[#57564]
+* Fixes chart resize after browser refresh {pull}57578[#57578]
+* Fixes hiding date picker for settings pages {pull}57544[#57544]
+Management::
+* Allows support for nested multi-fields {pull}58203[#58203]
+* Fixes performance bottleneck for large JSON payloads {pull}57668[#57668]
+* Fixes filter deprecations search filter {pull}57541[#57541]
+Maps::
+* Sets filter.meta.key to geoFieldName so query passes filterMatchesIndex when ignoreFilterIfFieldNotInIndex is true {pull}56692[#56692]
+* Fixes document source top hits split by scripted field {pull}57481[#57481]
+* Only request field in docvalue_fields when the field supports doc values {pull}57372[#57372]
+Monitoring::
+* Fixes issue when index pattern has no fields {pull}58242[#58242]
+* Fixes inaccuracies in Logstash pipeline listing metrics {pull}55868[#55868]
+Platform::
+* Limits fetching index patterns {pull}56603[#56603]
+* Fixes browser date format {pull}57714[#57714]
+* Prepends basePath in getUrlForApp {pull}57316[#57316]
+* Uses app id instead of pluginId to generate navlink from legacy apps {pull}57542[#57542]
+* Retries ES API calls that fail with 410/Gone to prevent {kib} from crashing at startup {pull}56950[#56950]
+* Removes injected reference from home app {pull}57836[#57836]
+Security::
+* Logout should redirect to the login screen at the server base path {pull}56786[#56786]
+* Adds xpack.encryptedSavedObjects.encryptionKey to docker allow-list {pull}58291[#58291]
+* Fixes short url in spaces {pull}58313[#58313]
+SIEM::
+* Backend end-to-end tests {pull}57166[#57166]
+* Removes internal tags when copying signals from rules {pull}57744[#57744]
+* Fixes return codes where some were rule_id instead of id {pull}57939[#57939]
+* Fixes Host Details Events Table to only show events for specified Host {pull}57388[#57388]
+Uptime::
+* Uses scripted metric for snapshot calculation {pull}58247[#58247]
+Visualizations::
+* Fixes auto refresh in visualizations and Lens {pull}57667[#57667]
```

Comment:
Since this one applies to Lens as well, can we combine into one section "Lens & Visualizations" and include all 3 fixes?